### DISCUSSION TITLE
#【速看】【亮点】有人山寨莫妮卡大姐姐！！！！！！

### DISCUSSION THREAD

**Discussion by RickyIOVEYOU** [share121/inter-knot#427] [其他]
这个什么“玛丽莲梦露”的老太婆，一看就是在山寨我们的莫妮卡大姐姐！！！！_绳匠，三天内，我要这个阿姨的所有信息！！_
![1723233897268](https://github.com/user-attachments/assets/3028c4b8-ba5f-4463-8842-3d4a651a63a8)

**Comment by Ailian-Qiao** [comment]
玛丽莲·梦露（Marilyn Monroe，1926年6月1日—1962年8月5日） [9]，本名“诺玛·简·莫泰森” [9]，出生在[美国加利福尼亚州](https://baike.baidu.com/item/%E7%BE%8E%E5%9B%BD%E5%8A%A0%E5%88%A9%E7%A6%8F%E5%B0%BC%E4%BA%9A%E5%B7%9E/12759534?fromModule=lemma_inlink)[洛杉矶](https://baike.baidu.com/item/%E6%B4%9B%E6%9D%89%E7%9F%B6/214107?fromModule=lemma_inlink)市 [9]，美国女演员、制片人 [9-10]。
1948年，出演了电影处女作《斯库达，嚯！斯库达，嗨！》 [20]。1950年，出演了剧情片《[彗星美人](https://baike.baidu.com/item/%E5%BD%97%E6%98%9F%E7%BE%8E%E4%BA%BA/3139788?fromModule=lemma_inlink)》，展示了她的光采 [11]。1952年，她开始担任女主角 [11]。次年推出的两部影片《[绅士爱美人](https://baike.baidu.com/item/%E7%BB%85%E5%A3%AB%E7%88%B1%E7%BE%8E%E4%BA%BA/8440060?fromModule=lemma_inlink)》《[如何嫁给一个百万富翁](https://baike.baidu.com/item/%E5%A6%82%E4%BD%95%E5%AB%81%E7%BB%99%E4%B8%80%E4%B8%AA%E7%99%BE%E4%B8%87%E5%AF%8C%E7%BF%81/1805487?fromModule=lemma_inlink)》，确认了她世界上最大牌的电影明星的地位 [11-13]。1955年6月3日，主演的爱情喜剧片《[七年之痒](https://baike.baidu.com/item/%E4%B8%83%E5%B9%B4%E4%B9%8B%E7%97%92/815255?fromModule=lemma_inlink)》上映 [14]，凭借该片获得[第9届英国电影和电视艺术学院奖](https://baike.baidu.com/item/%E7%AC%AC9%E5%B1%8A%E8%8B%B1%E5%9B%BD%E7%94%B5%E5%BD%B1%E5%92%8C%E7%94%B5%E8%A7%86%E8%89%BA%E6%9C%AF%E5%AD%A6%E9%99%A2%E5%A5%96/53965430?fromModule=lemma_inlink)电影奖-最佳外国女演员提名 [1]。之后，她离开好莱坞，来到纽约师从著名的表演老师学习表演 [11]。1957年6月13日，主演的爱情喜剧片《[游龙戏凤](https://baike.baidu.com/item/%E6%B8%B8%E9%BE%99%E6%88%8F%E5%87%A4/1413141?fromModule=lemma_inlink)》首映 [28]，由此提名[第11届英国电影和电视艺术学院奖](https://baike.baidu.com/item/%E7%AC%AC11%E5%B1%8A%E8%8B%B1%E5%9B%BD%E7%94%B5%E5%BD%B1%E5%92%8C%E7%94%B5%E8%A7%86%E8%89%BA%E6%9C%AF%E5%AD%A6%E9%99%A2%E5%A5%96/53965439?fromModule=lemma_inlink)电影奖-最佳外国女演员 [2]。这期间，梦露成立了自己的制作公司 [11]。1959年3月29日，她制作的最成功的影片《[热情似火](https://baike.baidu.com/item/%E7%83%AD%E6%83%85%E4%BC%BC%E7%81%AB/3102247?fromModule=lemma_inlink)》上映 [11] [15]，迎来她演艺生涯的顶峰 [16]，并凭借这部出色的电影获得了她演艺生涯中的最高肯定：金球奖最佳喜剧女主角 [16]。1961年2月1日，主演的西部爱情电影《[不合时宜的人](https://baike.baidu.com/item/%E4%B8%8D%E5%90%88%E6%97%B6%E5%AE%9C%E7%9A%84%E4%BA%BA/241204?fromModule=lemma_inlink)》首映 [17]，这是她最后一部完整的作品 [16]。次年，主演的喜剧短片《[濒于崩溃](https://baike.baidu.com/item/%E6%BF%92%E4%BA%8E%E5%B4%A9%E6%BA%83/476537?fromModule=lemma_inlink)》上映 [18]，这是她的最后一部电影 [16]。1962年8月5日，玛丽莲·梦露在洛杉矶布莱登木寓所的卧室内被发现去世，终年36岁 [19]。1999年，被美国电影学会评选为“百年来最伟大的女演员”第6名 [20]。
